src/screens/WhotoFollow/index.js

```diff
@@ -152,40 +152,7 @@ const WhotoFollow = () => {
       });
   }, []);
 
-  const register = async () => {
-    setFetchRegister(true);
-    Analytics.logEvent('onb_select_follows_btn_add', {
-      onb_whofollow_users_selected: followed
-    });
-    const data = {
-      users: {
-        username: usersState.username,
-        human_id: usersState.userId,
-        country_code: usersState.countryCode,
-        // human_id: randomString(16),
-        // country_code: 'US',
-        profile_pic_path: usersState.photoUrl,
-        status: 'A'
-      },
-      local_community: localCommunity.local_community,
-      topics: topics.topics,
-      follows: followed,
-      follow_source: 'onboarding'
-    };
-
-    const profilePic = usersState?.photoUrl;
-    if (profilePic && profilePic !== DEFAULT_PROFILE_PIC_PATH) {
-      try {
-        const uploadedImageUrl = await ImageUtils.uploadImageWithoutAuth(
-          data?.users?.profile_pic_path
-        );
-        data.users.profile_pic_path = uploadedImageUrl?.data?.url;
-        console.log('uploadedImageUrl', uploadedImageUrl);
-      } catch (e) {
-        console.log('error upload', e);
-      }
-    }
-
+  const registerAction = (data) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `registerAction` function is missing error handling. If the `registerUser` promise rejects, it will throw an unhandled promise rejection. It's a good practice to always handle promise rejections.
```diff
-    registerUser(data)
-      .then(async (res) => {
-        setFetchRegister(false);
+    registerUser(data)
+      .then(async (res) => {
+        setFetchRegister(false);
+      })
+      .catch((error) => {
+        console.error('Error during registration:', error);
+        // Handle the error appropriately here
+      });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
Fixed the feedback